packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -694,6 +694,56 @@ export = {
 
       test.done();
     },
+    'with startBatchBuild option'(test: Test) {
+      const stack = new cdk.Stack();
+
+      new codebuild.Project(stack, 'Project', {
+        source: codebuild.Source.gitHub({
+          owner: 'testowner',
+          repo: 'testrepo',
+          cloneDepth: 3,
+          fetchSubmodules: true,
+          webhook: true,
+          startBatchBuild: true,
+          reportBuildStatus: false,
+          webhookFilters: [
+            codebuild.FilterGroup.inEventOf(codebuild.EventAction.PUSH).andTagIsNot('stable'),
+            codebuild.FilterGroup.inEventOf(codebuild.EventAction.PULL_REQUEST_REOPENED).andBaseBranchIs('master'),
+          ],
+        }),
+      });
+
+      expect(stack).to(haveResource('AWS::CodeBuild::Project', {
+        Source: {
+          Type: 'GITHUB',
+          Location: 'https://github.com/testowner/testrepo.git',
+          ReportBuildStatus: false,
+          GitCloneDepth: 3,
+          GitSubmodulesConfig: {
+            FetchSubmodules: true,
+          },
+        },
+      }));
```

Comment:
I would remove this assertion, the test is not really about this part.

Reply:
yep. done